functions/go/Makefile

```diff
@@ -117,3 +118,9 @@ func-build: func-verify
 func-push:
 	@echo Pushing image $(CURRENT_FUNCTION)...
 	../../scripts/go-function-release.sh push curated
+
+generate:
+	GO111MODULE=on go get -v github.com/GoogleContainerTools/kpt/mdtogo
+	$(GOBIN)/mdtogo search-replace search-replace/generated --license=none --strategy=cmdDocs
```

Comment:
This is not consistent with the rest of the Makefile.
Ideally we only need to add a function name to this list (https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/1223aa9fa879d8c26c398bee3ef6d394a08002d1/functions/go/Makefile#L20-L26) instead of 2 places.

Reply:
Thanks for the pointer. Added new commit. https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/196/commits/b1398f7e8647d7fbea1a766573db9d62c3700c87